.github/workflows/workflow.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
         env:
           TESTCONTAINERS_RYUK_DISABLED: true
       - name: Build and Publish Docker image
-        if: github.ref == 'refs/heads/main'
+        if: ${{ github.ref == 'refs/heads/main' || github.ref == 'refs/heads/DIGIHOT-581-Statistikk-om-tidsbruk' }}
```

Comment:
Kan bruke on.push.branches så slipper man å skrive inn flere plasser. (Og beholde if (ref==main) på de stegene som kun skal gjelde for main. 
E.g. https://github.com/navikt/hm-soknad-api/blob/master/.github/workflows/deploy-dev.yml#L11

Reply:
Enig, vurderte det. Men det var sånn med IF fra starten og det var ikke riktig tidspunkt å bli distraert med det.